cmd/tsconnect/wasm/wasm_js.go

```diff
@@ -124,7 +124,7 @@ func newIPN(jsConfig js.Value) map[string]any {
 
 	logid := lpc.PublicID.String()
 	srv := ipnserver.New(logf, logid)
-	lb, err := ipnlocal.NewLocalBackend(logf, logid, store, "wasm", dialer, eng, controlclient.LoginEphemeral)
+	lb, err := ipnlocal.NewLocalBackend(logf, logid, store, "", dialer, eng, controlclient.LoginEphemeral)
```

Comment:
I think this might be the last user of this field, we can probably just drop it from `NewLocalBackend`

Reply:
Good point, added to my Fixit Friday cleanup list.